webmessaging/message-channels/close.html

```diff
@@ -71,4 +71,10 @@
     c.port2.postMessage('DONE');
   }, 'Close in onmessage should not cancel inflight messages.');
 
+test(() => {
+    const c = new MessageChannel();
+    c.port1.close();
+    assert_throws("DataCloneError", () => self.postMessage(null, "*", [c.port1]));
+    self.postMessage(null, "*", [c.port2]);
+}, "close() detaches a MessageChannel (but not the one its entangled with)");
 </script>
```

Comment:
you'll want to at least update the "Message sent from transferred closed port should not arrive." test above, since that one won't be valid anymore. Not sure if there are other tests that assume/verify the previous behavior as well.